prepend module load requirement

milljm opened the pull request:
Adding the necessary requirement on HPC systems when using versioner.py

Closes #29828

Commit:
prepend module load requirement

Adding the necessary requirement on HPC systems when using versioner.py

Closes #29828

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2676883/), step Docs: sync website on 9c91f64 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/29829/site)

This comment will be updated on new commits.

Commit:
move from .org address

Sometimes the .org address just don't work. Switch to the address
that does 100% of the time.
